### ISSUE TITLE
switching who is talking locally in chat

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#108]
allow quick switching between multiple people in the same game client, for cases where more than one are playing at once but taking turns. should be able to favorite and swap between combinations of player sprite and display name as well as any other relevant information that faces the other players
